osgp/shared/osgp-kafka-logger/src/main/java/org/opensmartgridplatform/kafka/logging/CountDownKafkaLogger.java

```diff
@@ -0,0 +1,56 @@
+/**
```

Comment:
if we would like to use this class in the existing consumer test in closed source, I think it should be moved to the project shared.